serializations.md

```diff
@@ -480,3 +481,23 @@ The channel offchain transaction is not included directly in the transaction tre
 , <closes_at>       , int()
 ]
 ```
+
+#### Proof of inclusion on state trees (POI)
+```
+[ {<accounts>  , [{binary(), [{binary(), [binary()]}]}]}
```

Comment:
The division of (normative) type (here) and (informative) label (below), plus usage of `,` instead of documented `::`, makes it difficult to parse&match for the reader. May I suggest unifying the two:
```
[ {<accounts>  :: [<proof_of_inclusion> :: {<root_hash> :: binary(), [{<mpt_hash> :: binary(), <mpt_value> :: [binary()]}]}]}
, {<calls>     :: [<proof_of_inclusion> :: {<root_hash> :: binary(), [{<mpt_hash> :: binary(), <mpt_value> :: [binary()]}]}]}
, {<channels>  :: [<proof_of_inclusion> :: {<root_hash> :: binary(), [{<mpt_hash> :: binary(), <mpt_value> :: [binary()]}]}]}
, {<contracts> :: [<proof_of_inclusion> :: {<root_hash> :: binary(), [{<mpt_hash> :: binary(), <mpt_value> :: [binary()]}]}]}
, {<ns>        :: [<proof_of_inclusion> :: {<root_hash> :: binary(), [{<mpt_hash> :: binary(), <mpt_value> :: [binary()]}]}]}
, {<oracles>   :: [<proof_of_inclusion> :: {<root_hash> :: binary(), [{<mpt_hash> :: binary(), <mpt_value> :: [binary()]}]}]}
]
```